cache_invalidator/Makefile

```diff
@@ -0,0 +1,9 @@
+CC=gcc
```

Comment:
I don't know about the project's perspective, but from a generic Makefile perspective, this would be a little more idiomatic:
```Makefile
CFLAGS = -fpic
LDFLAGS = -shared
LDLIBS = -lhiredis

libcache_invalidator.so: cache_invalidator.o
	$(CC) $(LDFLAGS) $< $(LOADLIBES) $(LDLIBS) -o $@

install: libcache_invalidator.so
	cp $< /usr/local/lib/
```

You could also go the other direction if this is a one-off:
```Makefile
libcache_invalidator.so: cache_invalidator.c
	gcc -fpic -shared -o $@ -lhiredis $<
install: libcache_invalidator.so
	cp $< /usr/local/lib/
```